### PR TITLE
Revamping of the user interface

### PR DESCRIPTION
**Pull request by AwYiss**
Here is a proposal for a revamping of the user interface. 

The changes mainly affect the banks and accounts lists, which are now located right beneath the app name, and displayed as dropdowns instead of mere lists. 

Besides, each bank or payment intermediary now has a logo, so the sidebar now has a less tedious look. Theses logos are grouped in a small sprite (~55ko).

The other small changes are mainly cosmetic:

* altered the appearance of panel and panel header
* switched from images to font icons for the menu items
* added icons to the wells at the top of the landing page

Here is a screenshot:

screenshot

### CODE REVIEW

File: server.coffee [hunk 1]
```diff
@@ -7,7 +7,7 @@ application = module.exports = (options, callback) ->
     options.name = 'Kresus'
     options.root ?= __dirname
     options.port ?= process.env.PORT || 9876
-    options.host = process.env.HOST || "127.0.0.1"
+    options.host = process.env.IP || "127.0.0.1"
```
Comment (bnjbvr) [inline server.coffee]: Please revert this change.

File: server/lib/accounts-manager.coffee [hunk 2]
```diff
@@ -52,8 +52,8 @@ TryMatchAccount = (target, accounts) ->
 
         # Remove spaces (e.g. Credit Mutuel would randomly add spaces in
         # account names) and lower case.
-        oldTitle = a.title.replace(/ /g, '').toLowerCase()
-        newTitle = target.title.replace(/ /g, '').toLowerCase()
+        oldTitle = a.title.toLowerCase()
+        newTitle = target.title.toLowerCase()
```
Comment (bnjbvr) [inline server/lib/accounts-manager.coffee]: Please revert this change.

File: client/css/style.css [hunk 3]
```diff
@@ -18,7 +36,26 @@ html, body, #main, #app {
 }
 
 body {
-    background: #f1f2f7;
+    font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-size: 13px;
+    color: #717171;
+    background-color: #f1f2f7;
+    -webkit-font-smoothing: antialiased;
```
Comment (bnjbvr) [inline client/css/style.css]: Can you please remove this non-standard property? (see also https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth)

**Comment by bnjbvr** [comment]
Thanks for the PR, I've just glanced at it and will need some time to take a look at the result. I think each bank's logo PNG has been added to the repository, although I would have expected only the sprite to be useful; can you remove these logos, please?

File: server/lib/accounts-manager.coffee (hunk 2 above)
Comment (AwYiss) [inline server/lib/accounts-manager.coffee]: I had an issue with those lines while setting up my online IDE and the change got commited by error. Sorry!

File: server.coffee (hunk 1 above)
Comment (AwYiss) [inline server.coffee]: I had an issue with this line while setting up my online IDE and the change got commited by error. Sorry!

File: client/css/style.css (hunk 3 above)
Comment (AwYiss) [inline client/css/style.css]: Done.

**Comment by AwYiss** [comment]
I was planning to add a step to the build script to automatically generate the sprite and the stylesheet from the images in the `static/images/banks` folder to make things easier when new banks are added.

If you don't feel like taking this path, though, I can remove the logos.

**Comment by ZeHiro** [comment]
Nice! But the advantage of raw list, is to have a brief overview of all the accounts at once, which is not possible anymore. Maybe it would be nice to have a new component to display the overview. This would be the default view when opening th app.

**Comment by bnjbvr** [comment]
@AwYiss, committing the images in the repo and generating the sprite / stylesheets from these sounds better (as long as images aren't used in the built frontend), thanks.

@ZeHiro, next version should have an overview screen! See #132 and don't hesitate to throw ideas there ;)

**Comment by bnjbvr** [comment]
Also @AwYiss told me on IRC that this was a work-in-progress patch. Please let me know when it's ready for review and merge!

**Comment by AwYiss** [comment]
All that's left is to change the look of items in the bank and account lists and in the settings section. I'll submit the patch in the next days.

**Comment by bnjbvr** [comment]
Hey @AwYiss, any news about this PR? I'm willing to refactor the frontend a bit, but I'd like to wait before your PR is done, to avoid painful rebasing conflicts. Or maybe you could squash your commits into a few ones (one per feature, ideally), and we'd check in this first part, and we can do the rest later?

**Comment by AwYiss** [comment]
Hi,

Sorry, last week was a busy one! I'll do my best to push the latest changes in the next days.

**Comment by bnjbvr** [comment]
Hey @AwYiss, any news about this PR? It looks quite good, and I am willing to change the frontend a lot, which implies splitting code into different files, renaming files, etc. So I'd like to merge your PR before starting these changes, unless you don't have time these days and it's ok for you to take it back later? Please let me know :)

**Comment by AwYiss** [comment]
I'm on it :smile:
